api/src/main/java/org/apache/cloudstack/api/command/user/iso/RegisterIsoCmd.java

```diff
@@ -177,6 +182,10 @@ public boolean isDirectDownload() {
         return directDownload == null ? false : directDownload;
     }
 
+    public Boolean isPasswordEnabled() {
+        return passwordEnabled;
```

Comment:
use a ternary operator like, `var == null ? false : var`

Reply:
Thanks for the reviews @rhtyd , @rafaelweingartner. I'm fixing this and pushing a new commit.

Reply:
@anuragaw don't see your new commit yet

Reply:
Apologies for the delay @rhtyd . I have pushed another commit and ran a test with blueorangutan.